website/sidebars.json

```diff
@@ -54,6 +43,24 @@
           "guides/rest-field-example",
           "guides/rest-route-example"
         ]
+      },
+      {
+        "type": "subcategory",
+        "label": "Columns",
```

Comment:
This feature name is so confusing, which Columns? It should probably be called `WP Table Columns` or something more descriptive